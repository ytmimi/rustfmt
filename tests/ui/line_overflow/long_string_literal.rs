// rustfmt-max_width: 50
// rustfmt-error_on_line_overflow: true
// rustfmt-error_on_unformatted: true

fn main() {
    println!("This is a really really long string that triggers an overflow warning")
}
