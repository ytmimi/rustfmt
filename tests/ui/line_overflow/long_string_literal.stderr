error[internal]: line formatted, but exceeded maximum width (maximum: 50 (see `max_width` option), found: 85)
 --> /Users/ytmimi/Documents/Projects/rust_projects/rustfmt/tests/ui/line_overflow/long_string_literal.rs:6:6:51
  |
6 |     println!("This is a really really long string that triggers an overflow warning")
  |                                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: set `error_on_unformatted = false` to suppress the warning against comments or string literals

warning: rustfmt has failed to format. See previous 1 errors.
